<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kafa</class>
 <widget class="QMainWindow" name="kafa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>605</width>
    <height>805</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>805</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Кафетерија</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="naslov">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Листа поруџбина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Одаберите врсту напитка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Црна кафа (50 динара)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Нес кафа (80 динара)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Капућино (100 динара)</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Додаци</string>
    </property>
    <widget class="QCheckBox" name="checkBoxSlagKrem">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Шлаг крем (30 динара)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPrelivCokolada">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Прелив од чоколаде (20 динара)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxKuglaSladoleda">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Кугла сладоледа (50 динара)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cenaLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цена напитка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="kolicinaLabela">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количина:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxKolicina">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="cenaNapitka">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="dinara">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>динара</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodajUListu">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Додај у листу</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="zakljuciPorudzbinu">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>740</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Закључи поруџбину</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
